pkg/abstractions/image/build.go

```diff
@@ -41,11 +41,12 @@ type BuildOpts struct {
 	BaseImageRegistry  string
 	BaseImageName      string
 	BaseImageTag       string
+	BaseImageCreds     string
```

Comment:
Will all image creds always be resolved to being a single string token?

Reply:
Yes, I believe so. Skopeo only has `--src-creds` or `--username` and `--password` flags.

How we get this username/password is where the details lie e.g. from an AWS account or from a GCP account.